### ISSUE TITLE
voila>= 0.2.0 raises ValueError: not enough values to unpack 

### ISSUE THREAD

**Issue by pgellert** [voila-dashboards/voila#714]
## Problem:

I am getting a ValueError when running voila as a standalone application. It only comes up for voila versions >= 0.2.0.

```
Traceback (most recent call last):
  File "e:\voila-test\venv\lib\site-packages\tornado\web.py", line 1703, in _execute
    result = await result
  File "e:\voila-test\venv\lib\site-packages\tornado\web.py", line 2580, in get
    absolute_path = self.get_absolute_path(self.root, self.path)
  File "e:\voila-test\venv\lib\site-packages\voila\static_file_handler.py", line 56, in get_absolute_path
    template, static, relpath = path.split('/', 2)
ValueError: not enough values to unpack (expected 3, got 1)
```

## Steps to reproduce:
In a fresh virtual environment:
`pip install voila==0.2.2 jupyter`
`voila Untitled.ipynb`
, where Untitled.ipynb is a basic jupyter notebook printing hello world.

The error above is raised while loading the notebook in Chrome.

Platform: Windows
Python version: 3.6.6
Pip list:
```
Package             Version
------------------- -------
argon2-cffi         20.1.0
async-generator     1.10
attrs               20.2.0
backcall            0.2.0
bleach              3.2.1
cffi                1.14.3
colorama            0.4.3
decorator           4.4.2
defusedxml          0.6.0
entrypoints         0.3
importlib-metadata  2.0.0
ipykernel           5.3.4
ipython             7.16.1
ipython-genutils    0.2.0
ipywidgets          7.5.1
jedi                0.17.2
Jinja2              2.11.2
jsonschema          3.2.0
jupyter             1.0.0
jupyter-client      6.1.7
jupyter-console     6.2.0
jupyter-core        4.6.3
jupyter-server      0.3.0
jupyterlab-pygments 0.1.1
MarkupSafe          1.1.1
mistune             0.8.4
nbclient            0.5.0
nbconvert           6.0.5
nbformat            5.0.7
nest-asyncio        1.4.0
notebook            6.1.4
packaging           20.4
pandocfilters       1.4.2
parso               0.7.1
pickleshare         0.7.5
pip                 20.2.3
prometheus-client   0.8.0
prompt-toolkit      3.0.7
pycparser           2.20
Pygments            2.7.1
pyparsing           2.4.7
pyrsistent          0.17.3
python-dateutil     2.8.1
pywin32             228
pywinpty            0.5.7
pyzmq               19.0.2
qtconsole           4.7.7
QtPy                1.9.0
Send2Trash          1.5.0
setuptools          50.3.0
six                 1.15.0
terminado           0.9.1
testpath            0.4.4
tornado             6.0.4
traitlets           4.3.3
voila               0.2.2
wcwidth             0.2.5
webencodings        0.5.1
wheel               0.35.1
widgetsnbextension  3.5.1
zipp                3.2.0
```

**Comment by SylvainCorlay** [comment]
Thanks for reporting @pgellert, I am looking into it.

**Comment by rth** [comment]
Also spoke to another person who run into this. It would likely help to add CI on windows https://github.com/voila-dashboards/voila/issues/526

**Comment by SylvainCorlay** [comment]
@rth @pgellert this is an issue with the recently released tornado 6.

I am looking into it.

In the mean time, a workaround is to switch back to tornado 5.x.

**Comment by SylvainCorlay** [comment]
I pushed a fix in #716. Will be tagging a release shortly after this gets in.

**Comment by pgellert** [comment]
Thank you!
